from exceptions_1.exercises import case_4, case_6, case_7


def main():
    print('Start up')

    # supermarket = {
    #     'food': ['egs', 'milk']
    # }
    #
    # case_4(supermarket, 'furniture')
    # try:
    #     case_6()
    # except NotImplementedError as nie:
    #     print(f'Exception cought: {nie}')
    #
    # print('Finish')

    case_7()


if __name__ == '__main__':
    main()
